### PR TITLE
[Docs] Add multi-cluster support faq

### PR DESCRIPTION
**Pull request by SbloodyS**
related pr: https://github.com/apache/dolphinscheduler/issues/427

### CODE REVIEW

**Comment by SbloodyS** [comment]
@zhongjiajie Please review. Thanks.

File: docs/en-us/release/faq.md [hunk 1]
```diff
@@ -607,4 +607,15 @@ sed -i 's/Defaults    requirett/#Defaults    requirett/g' /etc/sudoers
 
 ---
 
+## Q：Support for multiple YARN clusters
```
Comment (zhongjiajie) [inline docs/en-us/release/faq.md:610]: Maybe using the keyword `Deploy` is more reasonable
```suggestion
## Q: Deploy for multiple YARN clusters
```

File: docs/en-us/release/faq.md [hunk 2]
```diff
@@ -607,4 +607,15 @@ sed -i 's/Defaults    requirett/#Defaults    requirett/g' /etc/sudoers
...
+A：By deploying different worker in different yarn clusters，the steps are as follows(eg: AWS EMR):
+   
+   1. Deploying the worker server on the master node of the EMR cluster
+   
+   2. Changing yarn.application.status.address to current emr's yarn url in the conf/common.properties
```
Comment (zhongjiajie) [inline docs/en-us/release/faq.md:615]: We better use "``" hint configure file or code
```suggestion
   2. Changing yarn.application.status.address to current emr's yarn url in the `conf/common.properties`
```

File: docs/en-us/release/faq.md [hunk 3]
```diff
@@ -607,4 +607,15 @@ sed -i 's/Defaults    requirett/#Defaults    requirett/g' /etc/sudoers
...
+   
+   3. Execute command bin/dolphinscheduler-daemon.sh start worker-server and bin/dolphinscheduler-daemon.sh start logger-server to start worker-server and logger-server
```
Comment (zhongjiajie) [inline docs/en-us/release/faq.md:617]:
```suggestion
   3. Execute command `bin/dolphinscheduler-daemon.sh start worker-server` and `bin/dolphinscheduler-daemon.sh start logger-server` to start worker-server and logger-server
```

**Review by zhongjiajie — COMMENTED** [review]
I add some suggestion in English version, same as Zh version doc

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks

**Comment by zhongjiajie** [comment]
@SbloodyS What is the minimum version supported by this feature? Maybe we should add it to the documentation

**Comment by SbloodyS** [comment]
> @SbloodyS What is the minimum version supported by this feature? Maybe we should add it to the documentation

I started using it like this from 1.3.5-release. It seems that the versions we are still maintaining all support it.

**Comment by zhongjiajie** [comment]
Thanks @SbloodyS for clarification. In this case, we could keep it in old and do not without adding version hint 👍

**Pull request merged**